backend/LeetWars.Core/LeetWars.Core.BLL/MappingProfiles/LanguageProfile.cs

```diff
@@ -8,6 +8,7 @@ public class LanguageProfile : Profile
     {
         public LanguageProfile()
         {
+            CreateMap<Language, LanguageDto>();
```

Comment:
This is a redundant call, ReverseMap() is enough
https://dotnettutorials.net/lesson/reverse-mapping-using-automapper/